src/routes/api/collabs/[id]/picks/[pick_id]/index.ts

```diff
@@ -153,6 +199,8 @@ export const del: RequestHandler = async ({ request, params }) => {
 	const decodedUser = Jwt.decode(cookies['user_id']);
 	const userId = decodedUser['user_id'] as string;
 
+	const reason = request.headers.get('Reason');
```

Comment:
The body is available through `request.json()`, which will return the exact JSON you input in axios.

Reply:
Oh Ok! Made the variable return the reason from the body instead. Thanks!